styling-css-modules-sample/renderer.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { LensRendererExtension } from "@k8slens/extensions";
+import { ExampleIcon, ExamplePage } from "./page";
+
+export default class ModulesStylingExtension extends LensRendererExtension {
+  clusterPages = [
+    {
+      path: "/css-modules-styling-sample",
```

Comment:
API is changed, now page-registration and page-menu item splitted.
Working example:
```
  clusterPages = [
    {
      id: Symbol("example-extension"),
      components: {
        Page: () => <ExamplePage extension={this}/>,
      }
    }
  ]

  clusterPageMenus = [
    {
      id: this.clusterPages[0].id,
      title: "Hello World",
      components: {
        Icon: ExampleIcon
      }
    }
  ]
```

Reply:
Starting the app with described changes and got
```
Uncaught TypeError: Expected "name" to match "[^\/#\?]+?", but got "@k8slens/styling-css-modules-sample"
```
with app crash. Something with id?

Reply:
Upgrade to:
```
"@k8slens/extensions": "^4.0.0-beta.1",
```

Reply:
Adding a `url` doesn't help either:
```
clusterPages = [
    {
      id: "extension-example",
      components: {
        Page: () => <ExamplePage extension={this}/>,
      }
    }
  ]

  clusterPageMenus = [
    {
      id: this.clusterPages[0].id, // must be the same as in page registration to be visible in sidebar
      url: `/${this.clusterPages[0].id}`,
      title: "Hello World",
      components: {
        Icon: ExampleIcon
      }
    }
  ]
```

Reply:
Ok, probably better to wait fixes from https://github.com/lensapp/lens/pull/1364